src/pages/de/reference/configuration-reference.md

```diff
@@ -0,0 +1,372 @@
+{
+  markdown: {
+    // Beispiel: Alle Entwürfe in den endgültigen Build einbeziehen
+    drafts: true,
+  }
+}
+```
+
+
+### markdown.shikiConfig
+
+<p>
+
+**Typ:** `ShikiConfig`
+</p>
+
+Shiki Konfigurationspptionen. Siehe [Markdown Konfiguration Docs](https://docs.astro.build/en/guides/markdown-content/#shiki-configuration).
+
+
+### markdown.syntaxHighlight
+
+<p>
+
+**Typ:** `'shiki' | 'prism' | false`<br>
+**Standard:** `shiki`
+</p>
+
+Legt fest welcher Syntax-Highlighter zu verwenden ist, wenn überhaupt.
+- `shiki` - benutze die [Shiki](https://github.com/shikijs/shiki) Syntaxhervorhebung
+- `prism` - benutze die [Prism](https://prismjs.com/) Syntaxhervorhebung
+- `false` - keine Syntaxhervorhebung anwenden
+
+```js
+{
+  markdown: {
+    // Beispiel: Umschalten auf Prism zur Syntaxhervorhebung in Markdown
+    syntaxHighlight: 'prism',
+  }
+}
+```
+
+
+### markdown.remarkPlugins
+
+<p>
+
+**Typ:** `Array.<Plugin>`
+</p>
+
+Übergibt ein benutzerdefiniertes [Remark](https://github.com/remarkjs/remark) Plugin, um die Art und Weise, wie dein Markdown erstellt wird, anzupassen.
+
+**Anmerkung:** Die Aktivierung von benutzerdefinierten `remarkPlugins` oder `rehypePlugins` entfernt Astros native Unterstützung für [GitHub-flavored Markdown](https://github.github.com/gfm/), [Footnotes](https://github.com/remarkjs/remark-footnotes) Syntax, [Smartypants](https://github.com/silvenon/remark-smartypants). Du musst diese Plugins explizit in deine Datei `astro.config.mjs` aufnehmen, falls gewünscht.
+
+```js
+{
+  markdown: {
+    // Beispiel: Die von Astro verwendeten remark Standard-Plugins
+    remarkPlugins: ['remark-code-titles', ['rehype-autolink-headings', { behavior: 'prepend' }]],
+  },
+};
+```
+
+
+### markdown.rehypePlugins
+
+<p>
+
+**Typ:** `Array.<Plugin>`
+</p>
+
+Übergbt eine benutzerdefinierte [Rehype](https://github.com/remarkjs/remark-rehype) Plugin, um die Erstellung von Markdown anzupassen.
+
+**Anmerkung:** Die Aktivierung von benutzerdefinierten `remarkPlugins` oder `rehypePlugins` entfernt Astros native Unterstützung für [GitHub-flavored Markdown](https://github.github.com/gfm/), [Footnotes](https://github.com/remarkjs/remark-footnotes) Syntax, [Smartypants](https://github.com/silvenon/remark-smartypants). Du musst diese Plugins explizit in deine Datei `astro.config.mjs` aufnehmen, falls gewünscht.
+
+```js
+{
+  markdown: {
+    // Beispiel: Die von Astro verwendeten rehype Standard-Plugins
+    rehypePlugins: [['rehype-toc', { headings: ['h2', 'h3'] }], [addClasses, { 'h1,h2,h3': 'title' }], 'rehype-slug'],
+  },
+};
+```
+
+
+## Integrationen
+
+Erweitere Astro mit benutzerdefinierten Integrationen. Integrationen sind deine zentrale Anlaufstelle für das Hinzufügen von Framework-Unterstützung (wie Solid.js), neuen Funktionen (wie Sitemaps) und neuen Bibliotheken (wie Partytown und Tailwind).
+
+Lese unseren [Integrations-Leitfaden](/de/guides/integrations-guide/), um die ersten Schritte mit Astro-Integrationen zu gehen.
```

Comment:
```suggestion
Erweitere Astro mit benutzerdefinierten Integrationen. Integrationen sind deine zentrale Anlaufstelle für das Hinzufügen von UI-Framework-Unterstützungen (wie Solid.js), neuen Funktionen (wie Sitemaps) und neuen Bibliotheken (wie Partytown und Tailwind).

Lies unseren [Integrations-Leitfaden](/de/guides/integrations-guide/) für Hilfe bei der Einrichtung von Astro-Integrationen.
```

Reply:
- This is about frontend/javascript frameworks - not UI related.
- The mentioned guide should introduce the reader to the topic, it's meant to support them in their "first steps". So, I'd rather want to choose something like this. Good catch with the initial word though 😅

Reply:
- I picked this wording to match all other places in the docs where we reference frontend frameworks like Solid, React etc. We're calling them "UI-Frameworks" everywhere. :) Some examples:
  - https://docs.astro.build/de/install/auto/#installiere-offizielle-integrationen-optional
    > Du hast nun die Möglichkeit, die Unterstützung für [zusätzliche UI-Frameworks](https://docs.astro.build/de/core-concepts/framework-components/) (React, Svelte, Vue, Solid, Preact, Lit), sowie andere offizielle Astro-Integrationen (Tailwind, Partytown, Sitemap) zu installieren.
  - https://docs.astro.build/de/core-concepts/partial-hydration/
    > Du kannst Komponenten aus beliebigen UI-Frameworks verwenden ([React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) und [Lit](https://lit.dev/)) und Astro wird diese während des Buildvorgangs in reinen HTML-Code umwandeln (“rendern”) und den JavaScript-Code entfernen.
- Regarding the "first steps" part, the main reason why I changed the wording was that I felt that the German translation seemed to put a bit too much emphasis on "going" the first steps, like actually talking a walk through the park together with your favorite integration ("die ersten Schritte mit Astro-Integrationen zu gehen"). I tried to come up with a different way of saying this, but wasn't happy with anything that came to mind, so I finally checked the referenced page to see what it contains: It describes how to set up the integrations. That's why I picked this wording because it describes the actual contents on of the target page. However, all that being said, if you like the old wording more, I'm also fine with that.

Reply:
You're right, I've improved the old wording a bit ;)

But for the UI part. . . I wonder why. And just to tell from this example; we don't have it in the English version. For my understanding, we distinguish between front-end frameworks (like all the JS ones, but also whatever programming language can be used for frontend logic) and UI frameworks (like Bootstrap, Tailwind, etc., which, combined with HTML, make up the UI the user can interfere with. That's the layer between programming logic and user - the User Interface.).

Reply:
Good points! I understand your reasoning and think we should definitely revisit this topic later to check if the way we used to translate the various integration types so far can be improved. Even to me as a web developer who's using this stuff every day, the differentiation between "front-end frameworks" and "UI frameworks" isn't really clear. One could argue that those terms overlap a fair bit. Let's focus on consistency first and have a nice discussion about an updated translation glossary soon!